Beheerlandschap.md

```diff
@@ -23,36 +25,17 @@ De relatie tussen deze omgevingen is gevisualiseerd in de onderstaande image:
+* Voor elke Referentie Implementatie die in GitHub is vervaardigd bestaat in hub.docker.com een docker file. Aangezien er voor elke API een Referentie component is, is er ook voor elke API een docker file. De docker files worden vanuit GitHub gegenereerd (pijl 2).<br/><br/>_Er zijn echter ook docker files voor de volgende componenten: vng-referentielijsten, demo-api en  gemma-zaken-docs._<-- Zijn hiervoor dan geen RI's?<br/><br/>Ook voor deze omgeving geldt dat deze wordt beheerd door API Beheer.<br/><br/>
+* API testvoorziening:<br/> 
+Hoewel de API testvoorziening op dit moment draait in de hub.docker.com omgeving en in de toekomst in de Haven omgeving hebben we deze hier toch als een aparte omgeving neergezet. Het is immers een omgeving die API Beheer apart moet beheren en configureren.<br/><br/>
+De testvoorziening maakt gebruik van de docker files die beschikbaar worden gesteld via hub.docker.com. Dit kan op 3 manieren. Ten eerste kan de ATV gebruik maken van de docker files (pijl 12) via ref.tst.vng.cloud (pijl 11), dit is de default. Het gevolg is dan wel dat alle data die in deze docker worden geplaatst zichtbaar ook is voor anderen. De kans dat hier junk bij zit is redelijk groot aangezien deze wijze ook gebruikt wordt voor het leren omgaan met de API's. De ATV kan echter ook direct gebruik maken van de docker files op hub.docker.com (pijl 8). Daarbij heeft men volledige controle over de data. Dit zal een gebruiker zelf moeten configureren. Tenslotte kan de ATV nog gebruik maken van docker files die lokaal bij de gebruiker waardoor men nog meer controle heeft (niet uitgewerkt in het overzicht). Zo kan men er voor kiezen om een eigen applicatie aan de ATV te hangen en zo de werking van deze applicatie te testen.<br/><br/>
+Bij elke API standaard worden ook de bijbehorende testscripts ontwikkeld. De bron van deze scripts staan in GitHub en deze scripts moeten in de ATV geconfigureerd worden (pijl 3).<br/><br/>
+**LET OP!** API Beheer beheert slechts de gebruikersadministratie en de configuratie van de testscripts van de API standaarden die zij beheert. API Beheer beheert dus niet de API testvoorziening zelf.<br/><br/>
+* ref.tst.vng.cloud/tokens is een tool waarmee een JWT token kan worden gegenereerd dat kan worden gebruikt in de ATV. Daartoe moet het in de ATV (pijl 9) en in ref.tst.vng.cloud (pijl 7) worden geconfigureerd. De configuratie in de ATV betreft een copy en past slag, de configuratie in ref.tst.vng.cloud is echter onderhevig aan authorisaties. Vandaar dat er voor gekozen is die rechten aan het tool toe te kennen. Configuratie kan dus alleen via het tool. API Beheer heeft verder geen rol bij het configureren. In de situatie dat men de ATV gebruikt met een directe koppeling naar Hub.docker.com of een lokale docker file kan men er overigens voor kiezen om een eigen token te gebruiken.<br/><br/>
```

Comment:
Volgens mij draait de docker container in een docker omgeving op de ATV. In deze docker omgeving kun je ook andere, schone docker containers opstarten. Eerlijk gezegd weet ik niet zeker of een gebruiker daar wel rechten heeft om zelf autorisaties in te stellen. @joeribekker ?

Reply:
@michielverhoef Wat is je vraag precies?

Reply:
Wanneer een gebruiker van de testvoorziening een nieuwe, schone docker container opstart (ipv te testen tegen de default referentieimplementatie waar iedereen op kan testen), hoe worden dan de juiste autorisaties ingesteld? Kun je dat zelf doen of zijn die al ingesteld of zit het nog weer anders?

Het gaat er dus niet om een eigen, lokale container te gebruiken bij het testen maar een nieuwe, schone docker container van dockerhub opstarten.

Reply:
Tja, dat is een kwestie van test data inladen. Daar moet de testvoorziening dan een UItje voor hebben.

Reply:
Testdata is wat anders dan autorisaties toch?